README.md

```diff
@@ -1,59 +1,41 @@
 # ens-test-env
 
-## How it works
-
-The testing environment used here is implemented in the stateless and stateful tests for ENS app. The environment consists of two parts: the Anvil Ethereum node, and the docker graph instance. Which environment type you use is dependent on your testing circumstances.
+`ens-test-env` runs Anvil and ENSNode for the purposes of testing ENS.
 
 ## Configuration
 
-There should be a file named `ens-test-env.config.js` in your project's root directory.
-You can add a JSDoc type import to import the config type.
+To run your scripts in the context of the `ens-test-env`, you must provide a config file to `ens-test-env`.
 
 ```js
 /**
  * @type {import('@ensdomains/ens-test-env').ENSTestEnvConfig}
  **/
 module.exports = {
   deployCommand: 'pnpm hardhat deploy',
-  archive: {
-    subgraphId: 'QmXxAE7Urtv6TPa8o8XmPwLVQNbH6r35hRKHP63udTxTNa',
-    epochTime: 1646894980,
-    blockNumber: 12066620,
-    baseUrl: 'https://storage.googleapis.com/ens-manager-build-data',
-    network: 'mainnet',
-  },
-  docker: {
-    file: './docker-compose.yml',
-    sudo: false,
-  },
-  ethereum: {
-    chain: {
-      chainId: 0,
-    },
-    fork: {
-      url: 'https://example.com',
-    }
-  },
-  graph: {
-    bypassLocal: false,
-  },
   scripts: [
     {
       command: 'example',
       name: 'example',
       prefixColor: 'blue.bold',
       cwd: path.resolve('./'),
       finishOnExit: true,
-      waitForGraph: true,
     },
   ],
-  paths: {
-    archives: './archives',
-    data: './data',
-  },
 }
 ```
 
+## Pull Docker Images
+
+To make the initial run faster, consider pulling the ENSNode Docker images separately:
+
+```bash
+docker pull docker.io/library/postgres:17
+docker pull ghcr.io/foundry-rs/foundry:stable
+docker pull ghcr.io/namehash/ensnode/ensindexer:latest
+docker pull ghcr.io/namehash/ensnode/ensrainbow:latest
+docker pull ghcr.io/ensdomains/ens-metadata-service:latest
```

Comment:
we could run those inside test-env I think, conditionally

since ens-test-env supports running without ENSNode, not all of it has to be pulled